packages/yew/src/context.rs

```diff
@@ -81,22 +81,32 @@ impl<T: Clone + PartialEq + 'static> Component for ContextProvider<T> {
     type Properties = ContextProviderProps<T>;
 
     fn create(ctx: &Context<Self>) -> Self {
+        let props = ctx.props();
         Self {
-            context: ctx.props().context.clone(),
+            children: props.children.clone(),
+            context: props.context.clone(),
```

Comment:
Wouldn't it be better to clone `ctx.props()` as it's an Rc?

Reply:
Not really. There's no way to move a value out of `Rc`, just like a shared reference. By cloning, we would just be adding the cost of the count increment

Reply:
Ah yes sorry - that would just clone Component::Properties anyway 😞